traceability/pkg/beater/apigeelogbeater.go

```diff
@@ -18,23 +17,24 @@ type customLogBeater struct {
 	logglyClient   *apigee.LogglyClient
 	eventProcessor *apigee.EventProcessor
 	client         beat.Client
-	eventChannel   chan []byte
+	// eventChannel   chan []byte
 }
 
 var bt *customLogBeater
-var logglyConfig *config.LogglyConfig
+
+// var logglyConfig *config.LogglyConfig
```

Comment:
Should we cleanup commented loggly related code ?

Reply:
I didn't want to clean it completely up, if we have to reenable it later.  But it is in the history if we need to.

Clean it then?

Reply:
maybe comment why you have it commented out?